### ISSUE TITLE
getInput replaces only first occurrence of a space character

### ISSUE THREAD

**Issue by manusa** [actions/toolkit#127]
getInput function in `@actions/core` package only replaces the first occurrence of a "space" character.

e.g. 'require some value' will be transformed to 'INPUT_REQUIRE_SOME VALUE' which is wrong.

```
export function getInput(name: string, options?: InputOptions): string {
  const val: string =
    process.env[`INPUT_${name.replace(' ', '_').toUpperCase()}`] || ''
  if (options && options.required && !val) {
    throw new Error(`Input required and not supplied: ${name}`)
  }
/* ... */
```

Following line https://github.com/actions/toolkit/blob/5fd70ca47a73ac3cc66ae853f17f93367742e6e3/packages/core/src/core.ts#L75 should probably be replaced by:
```
process.env[`INPUT_${name.replace(/ /g, '_').toUpperCase()}`] || ''
```

**Comment by bryanmacfarlane** [comment]
Fixed will be in next release

**Comment by manusa** [comment]
I confirm it's fixed and available in v1.1.1
